fizz/CMakeLists.txt

```diff
@@ -87,6 +87,9 @@ else()
   list(APPEND CMAKE_REQUIRED_INCLUDES ${LIBGFLAGS_INCLUDE_DIR})
 endif()
 
+find_package(ZLIB REQUIRED)
+list(APPEND FIZZ_SHINY_DEPENDENCIES ZLIB::ZLIB)
```

Comment:
This shouldn't be necessary. I don't think it is common practice to use `add_dependency` on an imported library. If you use `target_link_libraries(fizz PUBLIC ZLIB::ZLIB)` then in case `ZLIB::ZLIB` is actually a top level target (e.g. you are vendoring zlib in a monorepo), then the dependency is implicitly set.